arangosh/Dump/DumpFeature.cpp

```diff
@@ -591,7 +591,15 @@ void DumpFeature::collectOptions(std::shared_ptr<options::ProgramOptions> option
 
   options->addOption("--maskings", "file with maskings definition",
                      new StringParameter(&_options.maskingsFile))
-                     .setIntroducedIn(30322).setIntroducedIn(30402);
+                     .setIntroducedIn(30322)
+                     .setIntroducedIn(30402);
+
+  options->addOption("--compress-output",
+                     "compress files containing collection contents using gzip format",
+                     new BooleanParameter(&_options.useGzip))
+                     .setIntroducedIn(30323)
```

Comment:
Let's not backport this to 3.3 anymore. I think backporting to 3.4 is already enough.
```suggestion
```